core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -276,6 +280,15 @@ func (fm *FluxMonitor) IsHibernating() bool {
 // Close implements the job.Service interface. It stops this instance from
 // polling, cleaning up resources.
 func (fm *FluxMonitor) Close() error {
+	fm.mu.Lock()
+	// Exit if already in the process of closing
+	if fm.closing {
```

Comment:
hmm there might have been a ticket to migrate over all services to utils.StartStopOnce - e.g. in HeadTracker:
```
return ht.StopOnce("HeadTracker", func() error {
   ...
})
```

Reply:
Great! thanks I'll have a look